src/Components/Common/SearchBox/SearchBox.jsx

```diff
@@ -28,6 +37,55 @@ class SearchBox extends Component {
     this.setState({ filter: filterValue });
   };
 
+  /**
+   * submit the search filter
+   * @param {object} e
+   * @returns {void}
+   */
+  onSubmitSearch = e => {
+    e.preventDefault();
+    let url;
+    const { value } = this.props;
+    const { filter } = this.state;
+    const { onSearch } = this.props;
+    const { history } = this.props;
```

Comment:
@coolbeatz71 Destructuring from props could have been done once.

Reply:
@gracelungu thx for the feedback, I am fixing the issue ASAP